src/libaktualizr/logging/boost_logging.cc

```diff
@@ -18,9 +19,10 @@ int64_t get_curlopt_verbose() { return gLoggingThreshold <= boost::log::trivial:
 
 void logger_init() {
   gLoggingThreshold = boost::log::trivial::info;
-  logging::add_console_log(std::cout, boost::log::keywords::format = "%Message%",
+  logging::add_console_log(std::cout, boost::log::keywords::format = "[%TimeStamp%]: %Message%",
```

Comment:
I'm unsure about this, when logging to stdout it's either handled by systemd which does the timestamping or during debug, when we can use `ts` from [moreutils](https://joeyh.name/code/moreutils/): `aktualizr | ts "%F %H:%M:%.S"`.

If we decide to have more flexible logging builtin, storing timestamps in log files would make sense though.

Reply:
Yes, you are right, will be ugly in systemd case.